csharp-basics/exercises/TypesAndVariables/MoreVariablesAndPrinting/Program.cs

```diff
@@ -6,26 +6,26 @@ class Program
     {
         static void Main(string[] args)
         {
-            string myName, myEyes, myTeeth, myHair;
-            int myAge, myHeight, myWeight;
```

Comment:
Nomainot mainīgo nosaukumus zemāk nemainot tos šeit pie to deklarācijas, vecie nosaukumi nesanāk kā neizmantotas vērtības tagad?

Reply:
Vecās ir padzēstas no koda, tādēļ sarkanas pasvītrotas. Nestrādātu nekas ja būtu mainīgais divreiz deklarēts. :)